oauthlib/oauth2/rfc8628/clients/__init__.py

```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
```

Comment:
this 
# -*- coding: utf-8 -*- line is not needed anymore on python3+

Reply:
Done